source/events.html.erb

```diff
@@ -0,0 +1,95 @@
+---
+title: Events
+description: Insightful talks and networking opportunities.
+---
+
+<div class="events-hero hero-block content-block">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col-lg">
+        <div class="pr-md-6 mb-3">
+          <h1 class="title">Events</h1>
+          <p class="lead mb-md-6">
+            Insightful talks and networking opportunities.
+          </p>
+          <%= partial "partials/newsletter_form" %>
+        </div>
+      </div>
+      <div class="col-lg">
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="events content-block content-block-gray-100">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col-lg">
+        <ul class="events-list">
+          <% data.events.each do |event, info| %>
+            <li class="events-list__item card">
+              <% if info[:image] == true %>
+                <img src="<%= image_path("events/#{event}@2x.jpg") %>" class="events-list__item__image card-img-left" alt="" width="444" height="auto">
+                <% else %>
+                <img src="<%= image_path("events/placeholder@2x.jpg") %>" class="events-list__item__image card-img-left" alt="" width="444" height="auto">
+              <% end %>
+
+              <div class="card-body">
+                <h4 class="card-title"><%= info.title %></h4>
+                <h6 class="card-subtitle mb-2 text-muted"><%= info.date %> - <%= info.event_type %></h6>
+                <p class="card-text"><%= info.description.truncate(170) %></p>
+                <ul class="list-inline">
+                  <li class="list-inline-item">
+                    <span class="badge <%= "badge-primary" if info.label == 'Meetup' %> <%= "badge-success" if info.label == 'Conference' %> <%= "badge-dark" if info.label == 'Community Day' %>"><%= info.label %></span>
+                  </li>
+                  <% if info[:links] %>
+                    <% if info[:links].website %>
+                      <li class="list-inline-item align-middle">
+                        <a href="<%= info[:links].website %>" target="_blank" class="main-link">
+                          Website
+                        </a>
+                      </li>
+                    <% end %>
+                    <% if info[:links].youtube %>
+                      <li class="list-inline-item align-middle">
+                        <a href="<%= info[:links].youtube %>" target="_blank">
+                          Video
```

Comment:
When it's a conference Video doesn't work well because in the conference playlist linked there are several videos. Which other terms can we use here?

Reply:
You were right, `Video Playlist` sounds better, maybe.